src/layer.js

```diff
@@ -71,7 +71,7 @@ export default class Layer extends Component {
   feature = (props, id) => ({
     type: "Feature",
     geometry: this.geometry(props.coordinates),
-    properties: { id }
+    properties: props.properties
```

Comment:
Can you do something like :
```
    properties: {
      ...props.properties,
      id
    }
```

I am using the id for the click and the hover interaction on the feature.